package.json

```diff
@@ -18,7 +18,7 @@
     "sdk"
   ],
   "engines": {
-    "node": ">= 14.0.0"
+    "node": ">= 18.0.0"
```

Comment:
I believe we consider this as a breaking change. Please indicate it by including exclamation mark in commit title. 

Also, do you think we should keep it in a separate branch for now?

Reply:
I added the exclamation mark to the PR title, and yes, we should keep it in separate branch until the major release :D